executor/poolmgr/gp.go

```diff
@@ -471,7 +471,7 @@ func (gp *GenericPool) createPool() error {
 						},
 					},
 					Containers: []apiv1.Container{
-						{
+						fission.MergeContainerSpecs(gp.env.Spec.Runtime.Container, &apiv1.Container{
```

Comment:
Should a similar merge not be applied to `newdeploy` environment in the specs, or am I missing something?

Reply:
Done 👍